content/blog/2023-04-13-rse-on-mastodon.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "State of Research software engineers on Mastodon"
+
+[extra]
+
+author = "Richard Darst"
++++
+
+Many domains have migrated to Mastodon as of 2023.  Have Research
+Software Engineers?  This post examines the situation, based on [this
+post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)
+
+Summary: not much yet
+
+This post can be updated as we get more information/the situation
+evolves, please send pull requests.
+
+
+## RSE organizations
+
+* US RSE: <https://fosstodon.org/@us_rse>
+* DE RSE: <https://fosstodon.org/@de_rse@mastodon.social>
+* Nordic RSE: <https://fosstodon.org/@nordic_rse>
+* Society of RSE (UK/int): ??
+* ... ?
+
+
+## Other accounts to follow to seed your community
+
+TODO (should we join some other list instead of make a new one here?)
+
+
+## Common hashtags
+
+* **#RSEng and #RSE** have been seen, but #RSEng is preferred
+  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502) (same
+  as Twitter).  There doesn't seem to be much activity there.
+
+* **#HPC** has a fair community.
+
+* **#OpenScience** can be found, in addition to **ResearchData** and
+  **ResearchSoftware**.
+
+* We propose **#SciComp** for as a general "scientific computing" tag
+  which can partly overlap with the above, but is more general to
+  people interested in the computing part of science.
+
+* There is the [a.gup.pe](https://a.gup.pe/u/researchsoftware) group
+  "researchsoftware", which will boost any post tagging it, and can be
+  used as a more targeted distribution group than tags
+  ([ref](https://fosstodon.org/@mattasdata@qoto.org/110151591358979693)).
+  We could also get a listing at <https://fedi.directory/>.
+
+
+## RSE-focused Mastodon servers
+
+There don't seem to be any servers specifically focused on the RSE
+community.  Perhaps the closest one is the HPC community
+<https://mast.hpc.social>, and we are invited to take part there
+([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  Many
+of the repliers used <https://fosstodon.org>.
+
+One could ask if if's better to have a common server, or be more
+distributed among the communities of other scientists to spread the
+word.  There is a [list of academic/science related servers at
+fedi-science](https://fediscience.org/server-list.html).
+
+There could be room for some RSE organization to set up a
+RSE-focused server. (editor note: I, rkdarst, would somewhat prefer a
+"scicomp"-focused mastodon server to bring together data, computing,
+software, etc. people.)
```

Comment:
In my opinion, there is a better chance of an RSE-focused server being set up because there are actual RSE organisations. "Scientific computing" is a much more nebulous and dispersed domain, and I suspect the only people who would actually sign up to it would be RSEs anyway (hence why not just make an RSE instance).

Some next actions would be making a formal proposal to some or all of the RSE organisations. There is the international RSE council which would be a catch-all: https://researchsoftware.org/council.html

Reply:
Also "scientific computing" would not be a good fit for RSEs who work in social science, humanities, multidisciplinary projects, etc.

When thinking of running an instance you need to think about governance, moderation, sustainability, etc - attaching to existing an community or (group of) organisation(s) should help with this.

Reply:
I agree with the comments that this is too broad and we couldn't get someone to run it.  Commented to say this.

Reply:
The HPC community has https://hpc.social which includes https://mast.hpc.social/about, supported by organizational contributions,